analytics/correlation_category.md

```diff
@@ -0,0 +1,35 @@
+# Correlation by Category
+
+Calculates Correlation by Category.
+
+## Input Data
+Input data should contain categorical columns and numeric columns.
+
+  * Category Column - Categorical column that you want to see correlation.
+  * Measured By Column - key column to aggregate measure.
```

Comment:
Column that defines dimension to calculate correlation. For each group, measure is aggregated for each value of this column, forming a vector that represents the group. Then, correlations are calculated among these vectors. If Date/POSIXct column is specified, measure is aggregated for each period of time (day, week, etc.) that is specified.

Reply:
Changed as suggested.